libnavigation-ui/src/main/java/com/mapbox/navigation/ui/utils/CompareUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.ui.utils
+
+object CompareUtils {
```

Comment:
Should this be `internal`?

Reply:
done. this PR was already merged into master. Please see https://github.com/mapbox/mapbox-navigation-android/pull/2647